user/src/com/google/gwt/user/client/ui/CellPanel.java

```diff
@@ -72,6 +72,8 @@ public abstract class CellPanel extends ComplexPanel {
 
   public CellPanel() {
     table = DOM.createTable();
+    //aria role to indicate it is a plain layout table
```

Comment:
checkstyle is going to require a space after the `//` and before the comment text
```suggestion
    // aria role to indicate it is a plain layout table
```